src/v3/test/integration/authenticator-verification-okta-verify-totp.test.tsx

```diff
@@ -44,6 +44,30 @@ describe('authenticator-verification-okta-verify-totp', () => {
     );
   });
 
+  it('should send correct payload when totp is padded with spaces', async () => {
+    const {
+      authClient, user, findByTestId, findByText,
+    } = await setup({ mockResponse });
+
+    await findByText(/Enter a code/);
+
+    const submitButton = await findByText('Verify', { selector: 'button' });
+    const otpEle = await findByTestId('credentials.totp') as HTMLInputElement;
+
+    const totp = '123456';
+    const totpWithSpaces = `   ${totp}   `;
```

Comment:
I like that we're testing different formats of where the spaces are. Can we vary these even more by also including other expected whitespace we'd remove? (e.g. line breaks, tabs, non-breaking utf-8 space chars)

Maybe they can be mixed into the existing tests without needing new ones, so the `totpWithSpaces` would be `totpWithWhitespace` and have a mixture of whitespace in its value